swarm/api/http/server.go

```diff
@@ -942,3 +972,16 @@ func (lrw *loggingResponseWriter) WriteHeader(code int) {
 func isDecryptError(err error) bool {
 	return strings.Contains(err.Error(), api.ErrDecrypt.Error())
 }
+
+func ValidateContentTypeHeader(r *http.Request) (string, error) {
```

Comment:
@lmars what is your input on this?

Reply:
I'm not really sure what the purpose of this code is?

Reply:
validating incoming mime type on upload from header, to make sure that the input we're getting is correct before storing it in a manifest - just to make sure we are on par with IETF RFCs